### ISSUE TITLE
[Regression] Fastlane update to 2.96.0 takes too long

### ISSUE THREAD

**Issue by angly-cat** [fastlane/fastlane#12584]
### Question Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Question Subject
Fastlane updating time.

#### Question Description
In my CI I use Docker containers with old versions of `fastlane`, which just updates itself as the first step of each of building scripts.

The issue is that the time needed to update `fastlane` to `2.96.0` has increased drastically comparing to previous versions.

Previously, it took less than a minute to update `fastlane` on one of my linux machines:

```
[12:31:14]: Updating fastlane from 2.70.3 to 2.95.0... 🚀
[12:32:10]: Finished updating fastlane
```

Now, on the same machine it takes almost a half of an hour:

```
[08:03:59]: Updating fastlane from 2.70.3 to 2.96.0... 🚀
[08:31:07]: Finished updating fastlane
```

On Mac mini it takes even more (45 minutes):

```
[01:50:28]: Updating fastlane from 2.95.0 to 2.96.0... 🚀
[02:35:35]: Finished updating fastlane
```

In general, it was OK for me to wait for a minute or two for `fastlane` to update, instead of updating docker container images after each `fastlane` release. However, because of this, the total time of execution of my CI scripts increased a lot.

Question: is this long `fastlane` updating time a regression?

**Comment by joshdholtz** [comment]
@angly-cat The _fastlane_ gem itself only increased in size by maybe 1MB so it wouldn't be directly to fastlane at all. We did update to `google-api-client-0.21.2` which does come with a decent amount of documentation but that is the only that is taking a significantly long time for me when installing _fastlane_ from scratch. Installing `2.96.0` from scratch probably take around 3.5 minutes.

If you are doing this on CI, I would suggest disabling documentation if you aren't already? - https://stackoverflow.com/a/1386014/2464643

**Comment by angly-cat** [comment]
> I would suggest disabling documentation if you aren't already?

Thanks for suggestion. I didn't disable documentation, but I'll try doing so and then write back if it worked or not.

**Comment by zidge** [comment]
disabled documentation as suggested but the build still times out on circle.ci after 10 minutes.

**Comment by jereanon** [comment]
I'm also experiencing this issue. For circleCI 2.0 users, you can add the following to your configuration.

>no_output_timeout: 30m

Fastlane is now taking ~20 minutes to update! ouch!

> [09:00:01]: Updating fastlane from 2.81.0 to 2.96.1... 🚀
(eval):129: warning: character class has ']' without escape
(eval):134: warning: else without rescue is useless
(eval):138: warning: else without rescue is useless
(eval):145: warning: else without rescue is useless
(eval):148: warning: else without rescue is useless
[09:19:33]: Finished updating fastlane

**Comment by angly-cat** [comment]
In the end, I wasn't able to make `fastlane` respect `gemrc`, so I removed `fastlane update_fastlane` from `Fastfile` and added `gem install fastlane --no-document --quiet` as a command to my CI script, which dramatically reduced update time. From minute in the past (and dozens of minutes with `2.96.0`) to mere seconds.

Also, plugins specified in `Pluginfile` usually take ~minute to install, but after I started to install these gems explicitly (with `--no-document` param) without trusting `Pluginfile` to do that, 2 plugins installation time for me reduced from 2 minutes to seconds as well.
